api-module-library/ironclad/test/api.test.js

```diff
@@ -128,13 +140,26 @@ describe('Ironclad API class', () => {
             expect(response).to.have.property('isRevertibleToReview');
         });
 
-        it('should list all workflow approvals', async () => {
+        it.skip('should list all workflow approvals', async () => {
             const response = await api.listAllWorkflowApprovals(workflowID);
             expect(response).to.have.property('workflowId');
             expect(response).to.have.property('title');
             expect(response).to.have.property('approvalGroups');
             expect(response).to.have.property('roles');
         });
+
+        it('should create a workflow comment', async () => {
+            const body = {
+                creator: {
+                    type: 'email',
+                    email: 'projectteam@lefthook.com',
+                },
+                comment: 'Testing a comment',
+            };
+
+            const response = await api.createWorkflowComment(workflowID, body);
+            response.should.have.status(204);
```

Comment:
` Invalid Chai property: status`

Reply:
@JonathanEdMoore Hmm, this works on my machine for some reason

Reply:
Must be another global test library